package.json

```diff
@@ -116,10 +116,10 @@
     "gulp-util": "3.0.8",
     "imports-loader": "0.8.0",
     "istanbul-instrumenter-loader": "3.0.1",
-    "jasmine-core": "2.8.0",
+    "jasmine-core": "3.2.1",
```

Comment:
Can you also update the `@types/jasmine`?

Reply:
Currently doing this